app/screens/home/common/post/index.ts

```diff
@@ -3,10 +3,10 @@
 
 import withObservables from '@nozbe/with-observables';
```

Comment:
Maybe I am missing something, but this file looks like it is doing nothing.

Reply:
@koox00 Did you have the @koox00 Did you get a chance to look into this?

Reply:
@larkox , missed the comment somehow... 😅 looking at it now, thanks!!

Reply:
you are right! there is no need for that